x-pack/plugins/security_solution/public/common/components/event_details/event_details.tsx

```diff
@@ -469,23 +476,17 @@ const EventDetailsComponent: React.FC<Props> = ({
   );
 
   return (
-    <GuidedOnboardingTourStep
```

Comment:
We should only wrap only the target tab with the `GuidedOnboardingTourStep` to avoid adding extra attributes to other tabs in their DOM elements and cause the tour anchor locate incorrectly.

Reply:
<img width="2545" alt="Screenshot 2024-05-01 at 14 51 28" src="https://github.com/elastic/kibana/assets/6295984/234d64d6-ca4b-45b7-b644-4b9cf22dd430">